*: fix lint errors when un-ignoring build directives

otan opened the pull request:
In #59540, we unhid certain files behind a build tag, causing lint
issues. This commit rectifies these.

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/60712)
<!-- Reviewable:end -->

otan committed:
*: fix lint errors when un-ignoring build directives

In #59540, we unhid certain files behind a build tag, causing lint
issues. This commit rectifies these.

Release note: None

otan commented:
bors r=irfansharif
